stream_alert_cli/terraform/flow_logs.py

```diff
@@ -0,0 +1,51 @@
+"""
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from stream_alert_cli.logger import LOGGER_CLI
+
```

Comment:
Is pylint not complaining about only one line between imports and code 🤔 . I thought it barked if there weren't 2

Reply:
I'll autopep it